components/Footer/Footer.tsx

```diff
@@ -0,0 +1,64 @@
+import Link from "next/link";
+import React from "react";
+import { Logo } from "../Logo/Logo";
+
+export interface Link {
+  label: string;
+  route: string;
+}
+
+interface FooterLinkProps extends React.HTMLProps<HTMLLIElement> {
+  route: string;
+}
+
+interface FooterProps extends Partial<React.HTMLProps<HTMLElement>> {
+  links: Link[];
+}
+
+const focusClasses = "focus:shadow-md focus:border-white focus:border-[2px]";
```

Comment:
@alexxuss1996 I just saw [this frame](https://www.figma.com/file/FS9WsAYrmkESsmUAGzYdNo/Juguetear?node-id=442-524&t=XurkuaMsdLVhmk6p-0) in Figma, check it. There you will see how is supposed to be rendered the `<a>` when it has focus.
Maybe we could add `pb-1` on focus.
Oh, I just saw that according to that frame, the border width should be of 3px.